playwright-tests/storage-states/wallet-connected-profile.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
This doesn't appear to be used anywhere -- and it serves the same purpose as "wallet-connected"